README.md

```diff
@@ -29,15 +29,64 @@ $ npm install
 
 ## Running the app
 
-```bash
-# development
-$ npm run start
-
-# watch mode
-$ npm run start:dev
-
-# production mode
-$ npm run start:prod
+- Using Docker
+  - `docker-compose up` (first time run will perform build)
+    - you can force a fresh build with `--build`
+    - you can background (run without holding up your CLI) by using `--detach`
+- Without Docker
+
+  ```bash
+    # development
+    $ npm run start
+
+    # watch mode
+    $ npm run start:dev
+
+    # production mode
+    $ npm run start:prod
+  ```
+
+## Publishing Messages
+
+To publish events to the org-creator service to process run this snippet as a ts or js file.
+
+```javascript
+import { connect, Codec, JSONCodec } from 'nats';
+
+const sampleCreateRequestClaim = {
+  id: '2952901f-6e22-445b-af6a-fg92f54cf26b',
+  token: '<valid_claim_token>',
```

Comment:
I still think some explanation of how to get a valid claim token would be helpful (e.g. what function to call in iam-client-lib or how to manually create a jwt and sign or something)